packages/docs/content/migration-guides/introducing-a-css-reset.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Introducing a CSS reset
+order: 10
+---
+
+The design system's CSS reset is a collection of HTML element-specific styles that simplifies the development process while providing cross-browser sensible defaults to build from.
+
+Our reset CSS is bundled with the rest of the design system's styles, so to use it all you need to do is [import the design system's CSS](/getting-started/for-developers/#including-the-css) using your preferred method. Even if you haven't imported one of our components, your application should visually align with the design system.
+
+If you're curious to see what changes have been made, you can preview our reset styles in [Storybook](/storybook/?path=/story/reset-overview--overview). We'll break down the major changes for you to review below.
+
+## Global defaults
+
+The CSS reset introduces globally-defined defaults that will impact the entire application once the CSS. These defaults include (but aren't limited to):
+
+- `box-sizing: border-box` is set on every element — including `*::before` and `*::after`. This ensures that the declared width of element is never exceeded due to padding or border.
+- The `<html>` element has `scroll-behavior: smooth` set on `:focus-within`. This is a CSS-only solution that provides smooth anchor scrolling.
+- Animations, transitions, and smooth scroll are removed on all elements - including `*::before` and `*::after` for users who have `prefers-reduced-motion` enabled to respect their preferences.
+- The `<body>` element effectively replaces the `.ds-base` class, defining `font-family`, `font-size`, `line-height`, and `color`.
+  - In addition, `<body>` now sets `background-color` to a theme's specified background.
+- The `outline` is removed on elements with the `tabindex="-1"` declared. This is removes focus rings for non-interactive elements, e.g., when activating a skip link will then send focus to a `<main>` or heading element.
```

Comment:
```suggestion
- The `outline` is removed on elements with the `tabindex="-1"` declared. This is removes focus rings for non-interactive elements. For example, when activating a skip link it will usually send focus to a `<main>` or heading element.
```
This might make it easier to read and simplifies the sentence structure